content/docs/start/data-management/data-versioning.md

```diff
@@ -234,7 +234,7 @@ $ type %TEMP%\data.xml >> data\data.xml
 </details>
 
 ```dvc
-$ dvc add data/data.xml
+$ dvc commit data/data.xml
```

Comment:
Agree. Same as how you can `git add` or `git commit` changes but most of the time I think people go for `add` first (`commit` later). So it's OK to just mention `dvc add` in the Get Started, I think.

Reply:
I am preparing a lecture about version control and will be covering Git and DVC. 

After introducing Git, DVC will be discussed and I am going to mention that it behaves similiarly to Git. 
When I read data-versioning.md to get some ideas, I was confused that dvc add was used in this case but not dvc commit. Because I think for Git you would use  commit in this case and not add (?).

Maybe my confusion came about because I was thinking a lot about Git and didn't recognize that "adding" also "commits" in DVC. 
But maybe it helps beginners coming from Git to add a small sentence that, in contrast to Git, "adding" also "commits" in data-versioning.md?

(Anyway, there's nothing wrong with the documentation, so it's perfectly fine to leave it as is.)